src/journal-remote/journal-remote.c

```diff
@@ -777,21 +788,37 @@ static int setup_microhttpd_socket(RemoteServer *s,
         return setup_microhttpd_server(s, fd, key, cert, trust);
 }
 
+static int null_timerfd_handler(sd_event_source *event, uint64_t usec, void *userdata)
+{
+    return dispatch_http_event(event, 0, 0, userdata);
+}
+
 static int dispatch_http_event(sd_event_source *event,
                                int fd,
                                uint32_t revents,
                                void *userdata) {
         MHDDaemonWrapper *d = userdata;
         int r;
+        fd_set rd, w, e;
+        unsigned long long timeout = -1ULL;
 
         assert(d);
 
-        r = MHD_run(d->daemon);
+        MHD_get_fdset (d->daemon, &rd, &w, &e, 0);
```

Comment:
please follow coding style: no space between function name and opening `(`, also see CODING_STYLE

Reply:
also, we always check for functions returning error, and we should do here too